tests/js/spec/utils/replays/getCurrentUrl.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import getCurrentUrl from 'sentry/utils/replays/getCurrentUrl';
+import ReplayReader from 'sentry/utils/replays/replayReader';
+
+import {
+  createEventTransaction,
+  createReplayEvents,
+  createRrWebEvents,
+} from './mocks/events.fixture';
+
+const CURRENT_TIME_MS = 55114;
```

Comment:
Given this is a very specific time (and not necessarily "current" time"), I would just declare it in the test function that uses it, and give it an explicit name, e.g.:

```
const timeAfterNavigationEventOccurs = 55114;
```